exampleCourse/README.md

```diff
@@ -0,0 +1,5 @@
+# exampleCourse
+
+This directory contains a comprehensive example course that demonstrates many of the features of PrairieLearn. It is intended to be used as a reference for instructors and course developers.
+
+This course will be loaded into the PrairieLearn instance when you run PrairieLearn locally. More information about running PrairieLearn locally can be found on the docs in the [Installation guide](https://prairielearn.readthedocs.io/en/latest/installing/).
```

Comment:
(I'm sensing a pattern here. No need to capitalize words in the middle of the sentence like this.)

```suggestion
This course will be loaded into the PrairieLearn instance when you run PrairieLearn locally. More information about running PrairieLearn locally can be found on the docs in the [installation guide](https://prairielearn.readthedocs.io/en/latest/installing/).
```